strings/strings.ko-KR.json

```
@@ -2443,38 +2443,38 @@
   "tech_neutronium_housing_effect": "%0에서의 생활 공간 비용 감소",
   "tech_unification": "통일",
   "tech_unification_desc": "%0 통일",
-  "tech_unification_effect": "모든 국가 지배",
+  "tech_unification_effect": "전 세계를 당신의 통치 하에 통일할 계획을 세웁니다.",
   "tech_unification_effect2": "통일을 완료하려면 3개의 도시를 획득해야 합니다.",
   "tech_wc_conquest": "정복",
   "tech_wc_conquest_desc": "%0 정복",
   "tech_wc_conquest_desc_req": "모든 도시를 지배해야 합니다.",
   "tech_wc_conquest_effect": "강한 군사력으로 세계 정복",
   "tech_unification_warning": "통일 후에는 더 이상 전투가 없지만, 생산력은 25% 증가하고, 저장 용량은 2배 증가합니다.",
   "tech_unification_banana": "통일은 바나나 공화국 시나리오를 끝냅니다. 추가적인 바나나 스무디나 바나나 빙수 업적을 얻을 수 없게 될 것입니다.",
-  "tech_unite": "연합 국가",
-  "tech_unite_desc": "Unite your region of city states into a country.",
-  "tech_unite_effect": "Become one unified country, gain bonuses to storage and production.",
+  "tech_unite": "국가 통일",
+  "tech_unite_desc": "우리 지역에 있는 도시 국가들을 하나의 국가로 통일합니다.",
+  "tech_unite_effect": "하나의 통일된 국가가 됩니다. 저장 공간과 생산량 보너스를 얻습니다.",
   "tech_wc_morale": "문화 패권",
   "tech_wc_morale_desc": "%0 합병",
   "tech_wc_morale_desc_req": "행복도 %0% 필요",
-  "tech_wc_morale_effect": "높은 문화로 %0을/를 합병합니다.",
+  "tech_wc_morale_effect": "우리 국가의 우월한 문화로 %0을/를 점령합니다.",
```

take over의 의미상(인수, 인계 등) 평화적으로 지배권을 넘겨받는 것이어서 점령보다는 무력이 아닌 느낌이 드는 단어를 고르는 게 좋아 보입니다.

```suggestion
  "tech_wc_morale_effect": "우리의 우월한 문화를 바탕으로 %0을/를 확보합니다.",
```
확보는 임의로 고른 것이고 원래의 합병도 괜찮을 것 같고, money의 장악도 괜찮은 것 같습니다.